docs/tutorial.rst

```diff
@@ -394,8 +388,7 @@ It shows
 
 .. code-block:: bash
 
-  .
```

Comment:
I thought this `.` was needed for good indentation or something in the final HTML: https://github.com/citusdata/pg_auto_failover/commit/6efcbf2c207b42c049d772e4697d410e84859d9b

The same for the other dots that were removed.

Reply:
I have added non-breaking white space instead. They display nice in Emacs, and don't disturb the readers of the HTML, where we still have the intended markup.